Core/src/Upload/ResumableUploader.php

```diff
@@ -264,7 +264,7 @@ protected function getStatusResponse()
         $request = new Request(
             'PUT',
             $this->resumeUri,
-            ['Content-Range' => 'bytes */*']
+            ['Content-Range' => 'bytes */'.$this->data->getSize()]
```

Comment:
Can you please verify if this change affects any other products?
```suggestion
            ['Content-Range' => 'bytes */' . $this->data->getSize()]
```